app/client/cli/gov.go

```diff
@@ -76,5 +92,8 @@ func govCommands() []*cobra.Command {
 			},
 		},
 	}
+	for _, cmd := range cmds {
```

Comment:
@Olshansk this command seems tightly coupled to be specific to the actor case as the `actorCmdDef` option part of the `applySubcommandOptions` function which uses the `attachPwdFlagToSubcommands` relies on the actor type being defined - it might be better to refactor this logic to be generally applicable to arbitrary commands

Reply:
I changed the `applySubcommandOptions` to recieve only the options part of the `actorCmdDef` type so this can be used with the result of `attachPwdFlagToSubcommands` directly

Reply:
Change to `applySubcommandOptions` makes sense to me